packages/pwa-buildpack/lib/Utilities/serve.js

```diff
@@ -29,12 +30,23 @@ module.exports = async function serve(dirname) {
                     ...config.section('imageOptimizing'),
                     ...config.section('imageService')
                 });
+                if (process.env.ENABLE_EXPRESS_SERVER_COMPRESSION === 'true') {
+                    app.use(
+                        compression({
+                            threshold: 0
+                        })
+                    );
+                }
             }
         }
     );
 
     let envPort;
-    if (process.env.PORT) {
+    /**
+     * null and undefined are represented as strings in the env
+     * so we have to match using strings instead.
+     */
+    if (process.env.PORT !== 'null' && process.env.PORT !== 'undefined') {
```

Comment:
I was going to propose `parseInt` here, but I guess 0 is a valid port number. I'm fine with this, but a little type checking wouldn't hurt.

Reply:
Yeah you are right, 0 is a valid port number.